solutions/cf-1555B.mdx

```diff
@@ -0,0 +1,321 @@
+---
+id: cf-1555B
+source: CF 1555B
+title: B. Two Tables
+author: David Zhang
+---
+
+[Official Analysis (C++)](https://codeforces.com/blog/entry/93389)
+
+<Info title="Info/Tips">
+Oftentimes, problems will try to trick you. For example, since the judge is asking for a double, you would assume you may need to some advanced geometry or some diagonal movements, but a good rule of thumb is to always stay simple. As the most elegant and more importantly the fastest solutions are the shortest.
+</Info>
+
+## Explanation
+
+Let's call: 
+* $x_{1}$ and $x_{2}$ the leftmost and rightmost positions of the inital rectangle respectively
+* $y_{1}$ and $y_{2}$ the bottommost and topmost positions of the inital rectangle respectively
+* $w_{1}$ and $h_{1}$ the width and height of the inital rectangle (which can be calculated by calulating the difference in x and y)
+* $w_{2}$ and $h_{2}$ the width and height of the second rectangle respectively
+* $W$ and $H$ the total width and height of the room
+
+More importantly, there are 4 different places the second rectangle could be placed: left, right, top, bottom.
+For each place, we can calculate the correct x or y positions in order to just barely fit the second rectangle.
+
+Here's a diagram
+![](cf-1555B/PlacementsDiagram.png)
```

Comment:
maybe this instead?

<img width="476" alt="image" src="https://user-images.githubusercontent.com/81596991/160272347-bbb96e0d-a488-443d-9293-27585f378e68.png">

Reply:
looks cleaner